devicetypes/curb/curb-power-meter.src/curb-power-meter.groovy

```diff
@@ -0,0 +1,54 @@
+/**
+ *  Curb Power Meter
+ *
+ *  Copyright 2017 Curb
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+    definition(name: "CURB Power Meter", namespace: "curb", author: "Curb") {
```

Comment:
This line should be:

`definition(name: "CURB Power Meter", namespace: "curb", author: "Curb", ocfDeviceType: "x.com.st.d.energymeter") {`